ninja-core/src/main/java/ninja/NinjaDefault.java

```diff
@@ -477,9 +482,16 @@ private final String readNinjaVersion() {
     private final void showSplashScreenViaLogger() {
         
         String ninjaVersion = readNinjaVersion();
-        
-        // log Ninja splash screen
-        logger.info(NINJA_LOGO, ninjaVersion);
+
+        // log Ninja splash screen, from resources if available (so it can be overridden)
+        String NINJA_LOGO_LOCATION = "ninja/logo.txt";
+        try (InputStream is = this.getClass().getClassLoader().getResourceAsStream(NINJA_LOGO_LOCATION);) {
```

Comment:
Do you see a simple way to test it? To make sure the stuff keeps on working after we eg do some refactoring?